product-schema.json

```diff
@@ -0,0 +1,502 @@
+                                "SAX-PD",
+                                "Saxpath",
+                                "SCEA",
+                                "Sendmail",
+                                "SGI-B-1.0",
+                                "SGI-B-1.1",
+                                "SGI-B-2.0",
+                                "SimPL-2.0",
+                                "SISSL",
+                                "SISSL-1.2",
+                                "Sleepycat",
+                                "SMLNJ",
+                                "SMPPL",
+                                "SNIA",
+                                "Spencer-86",
+                                "Spencer-94",
+                                "Spencer-99",
+                                "SPL-1.0",
+                                "SugarCRM-1.1.3",
+                                "SWL",
+                                "TCL",
+                                "TMate",
+                                "TORQUE-1.1",
+                                "TOSL",
+                                "Unicode-TOU",
+                                "Unlicense",
+                                "UPL-1.0",
+                                "Vim",
+                                "VOSTROM",
+                                "VSL-1.0",
+                                "W3C",
+                                "W3C-19980720",
+                                "Watcom-1.0",
+                                "Wsuipa",
+                                "WTFPL",
+                                "X11",
+                                "Xerox",
+                                "XFree86-1.1",
+                                "xinetd",
+                                "Xnet",
+                                "xpp",
+                                "XSkat",
+                                "YPL-1.0",
+                                "YPL-1.1",
+                                "Zed",
+                                "Zend-2.0",
+                                "Zimbra-1.3",
+                                "Zimbra-1.4",
+                                "Zlib",
+                                "zlib-acknowledgement",
+                                "ZPL-1.1",
+                                "ZPL-2.0",
+                                "ZPL-2.1"]
+                    },
+                    "licenseURL": {
+                        "type": "string",
+                        "description": "Link to the license under which the Global Digital Pubic Good is released."
+                    }
+                }
+            }
+        },
+    	"SDGs": {
+          "type": "array",
+          "description": "List of Sustainable Development Goals that this Global Digital Public Good addresses.",
+          "minItems": 1,
+          "uniqueItems": true,
+          "items": {
+            "type": "integer",
+            "enum": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17]
```

Comment:
Should these include the names? I imagine that we will create a simple "schema creator" that feeds from this, so having more complete data could be useful. It could also make the schema files more readable on their own. Thoughts?

Reply:
In my opinion and in the spirit of keeping things simple, the numbers suffice as they convey the minimum and necessary information. Anyone can easily link the numbers to the actual names in their applications. Having a numeric array is also easier to validate, so that one doesn't get caught up with: `Zero Hunger` vs `Zero hunger` vs `ZERO HUNGER`.

Reply:
That's why the enum is important and would help validate. But if this is meant to stay internal integers are fine by me.

Reply:
Implemented in 3ae3d18